src/lib.rs

```diff
@@ -861,26 +908,35 @@ impl fmt::Debug for Client {
     }
 }
 
-struct Key {
+/// This struct represents a key used to sign requests to the ACME server
```

Comment:
"Key" can be fairly ambiguous in this context :-) Let's explicitly say it's the **account** private key.

```suggestion
/// This struct represents an account private key used to sign requests to the ACME server
```